scripts/cloudfoundry/bat-cdn/bat-cdn.rb

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env ruby
```

Comment:
Please add execute permission to the script

Reply:
done....